tests/ui/impl-trait/precise-capturing/bound-modifiers.rs

```diff
@@ -0,0 +1,25 @@
+//@ edition: 2021
+
+#![feature(precise_capturing)]
+
+fn polarity() -> impl Sized + ?use<> {}
+//~^ ERROR expected identifier, found keyword `use`
```

Comment:
I chose against generalizing `error_lt_bound_with_modifiers` for `use<>` precise capturing syntax. I guess I could? Do you think anyone would really ever write `?use<>` or `for<'a> use<'a>`??